docs/guides/cheatsheet/link_properties.rst

```diff
@@ -158,6 +158,53 @@ Querying
     },
   }
 
+.. note::
+
+    Specifying link properties of a computed backlink in your shape is
+    supported as of EdgeDB 3.0.
+
+    If you have this schema:
+
+    .. code-block:: sdl
+
+        type Person {
+          required name: str;
+          multi follows: Person {
```

Comment:
is link optional when using `multi link`? or is that a typo?

Reply:
`link` is optional in the new SDL syntax shipping with 3.0, except in the case of computed links.